scripts/generate_checksum.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHASUM_FILES="SHASUMS"
+FILE_PREFIX="eksctl-anywhere"
+
+DARWIN_TAR="darwin-amd64.tar.gz"
+LINUX_TAR="linux-amd64.tar.gz"
+
+release_manifest=$(curl --silent 'https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml')
+
+latest_release_version=$(echo $release_manifest | yq e '.spec.latestVersion' - -v)
+echo $latest_release_version
+
+latest_release=$(echo $release_manifest | yq e '.spec.releases[] | select(.version == "'$latest_release_version'")')
+echo $latest_release
+
+linux_sha256=$(echo $latest_release | yq e '.eksABinary.linux.sha256')
+darwin_sha256=$(echo $latest_release | yq e '.eksABinary.darwin.sha256')
+echo $linux_sha256
+
+linux_sha512=$(echo $latest_release | yq e '.eksABinary.linux.sha512')
+darwin_sha512=$(echo $latest_release | yq e '.eksABinary.darwin.sha512')
+
+mkdir -p $SHASUM_FILES
+cd $SHASUM_FILES
+
+SHA256_FILENAME="{FILE_PREFIX}-{latest_release_version}-sha256sums.txt"
+SHA512_FILENAME="{FILE_PREFIX}-{latest_release_version}-sha512sums.txt"
+
+echo "${linux_sha256} ${FILE_PREFIX}-${latest_release_version}-${LINUX_TAR}" >> SHA256_FILENAME
+echo "${darwin_sha256} ${FILE_PREFIX}-${latest_release_version}-${DARWIN_TAR}" >> SHA256_FILENAME
+
+echo "${linux_sha512} ${FILE_PREFIX}-${latest_release_version}-${LINUX_TAR}" >> SHA512_FILENAME
+echo "${darwin_sha512} ${FILE_PREFIX}-${latest_release_version}-${DARWIN_TAR}" >> SHA512_FILENAME
```

Comment:
Is everything after `>>` meant to be a variable? I know this is a draft so you're probably working out kinks.

Reply:
yeah I missed the `$` sign here, its updated now